src/cow-react/pages/Account/LockedGnoVesting/index.tsx

```diff
@@ -32,6 +31,7 @@ import { CurrencyAmount, Currency } from '@uniswap/sdk-core'
 import { getProviderErrorMessage, isRejectRequestProviderError } from 'utils/misc'
 import { claimAnalytics } from 'utils/analytics'
 import { ButtonSize } from 'theme'
+import { HelpCircle } from '@src/cow-react/modules/shared/dumb/HelpCircle'
```

Comment:
```suggestion
import { HelpCircle } from 'cow-react/modules/shared/dumb/HelpCircle'
```

Reply:
addressed in another PR. I only saw later